src/main/providers/launcher.provider.ts

```diff
@@ -30,16 +32,16 @@ export default class LauncherProvider {
 
     const launcher = new MinecraftLauncher({
       authentication: {
-        name: 'Player',
+        name: playerData.nick,
       },
       gameRoot: mcDir,
       assetsRoot: this.assetsDir,
       libraryRoot: this.librariesDir,
       nativesRoot: this.nativesDir,
       versionRoot: this.versionsDir,
       memory: {
-        min: 1024,
-        max: 2048,
+        min: parseInt(ramData.min),
+        max: parseInt(ramData.max),
```

Comment:
No need to parse int. It would be better if the memory configuration would directly give us an integer.